### PR TITLE
feat(frontend): new tailwind color

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Adding missing `black` color, will be later used in the Carousel indicators component.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default {
 			inherit: 'inherit',
 			transparent: 'transparent',
 			current: 'currentColor',
+			black: 'rgb(0, 0, 0)',
```
Comment (peterpeterparker) [inline tailwind.config.js:14]: You need this "color", using `color: initial` would not work out?

Comment (peterpeterparker) [reply tailwind.config.js:14]: Not against adding it nor strong opinion, really just asking.

Comment (DenysKarmazynDFINITY) [reply tailwind.config.js:14]: Unfortunately, `initial` doesn't work.

Comment (peterpeterparker) [reply tailwind.config.js:14]: 🥲

**Pull request merged**